portal-api/api-tests/v1/users/with-validate-password.api-tests.js

```diff
@@ -114,8 +119,9 @@ function itShouldReturnAnErrorIf(testCases) {
     async ({
       makeRequestWithModifiedPayload,
       expectResponseErrorToMatchExpected = (body) => expect(body.errors.errorList.password).toEqual(PASSWORD_ERROR),
+      middleware,
     }) => {
-      const { status, body } = await makeRequestWithModifiedPayload();
+      const { status, body } = await makeRequestWithModifiedPayload(middleware);
```

Comment:
Why is the middleware being passed as an argument, can a standard pattern be followed please.

Reply:
Removed argument.